### DISCUSSION TITLE
Per Sheet Settings

### DISCUSSION THREAD

**Discussion by idiotekque** [misthero/dnd5e-custom-skills#2] [Ideas]
Hey there! I'm a newbie to Foundry and trying to set up a semi-custom system that uses 5e and was grateful to find your module. Massive help for tweaking things a bit.

I was curious, though, if it might be possible (or if it already is somehow) to apply the edited abilities/skills through the module differently to different sheets? For what I'm trying to do, basically, there are character sheets that a bit more in line with standard 5e abilities and skills, but I also have a different system for vehicles and mechanical "actors", which would ideally have simplified ability scores and skills.

Unfortunately, I'm not sure what I can do to create any sort of different sheet for those cases. Hope the message isn't a bother, but I wasn't sure how to contact you otherwise. If you get a chance to reply, let me know. Thanks!

**Comment by misthero** [comment]
Thank you for your feedback, 
I appreciate the module is being useful for you, what you ask could be a nice enhancement.

I do not plan to develop new features in the near future, since I'm still refining and debugging and I'm doing this in my spare time. 
This module just came out and, even if the base is working, I need to make some change here and there to optimize everything.

I will keep this request here, for future reference or in case some other developers is intrested in adding this feature, pull requests are open.
